src/app/extensions/wishlists/shared/select-wishlist-modal/select-wishlist-modal.component.ts

```diff
@@ -231,4 +274,9 @@ export class SelectWishlistModalComponent implements OnInit, OnDestroy {
       ? 'account.wishlists.add_to_wishlist.confirmation'
       : 'account.wishlists.move_wishlist_item.confirmation';
   }
+
+  ngOnDestroy() {
+    this.destroy$.next();
+    this.destroy$.complete();
+  }
```

Comment:
This component has still a lot of lines of code. Consider to outsource the multiplefield form functionality in an extra component